string goalStatus
float32 wayPtEast
float32 wayPtNorth